### PR TITLE
Enable region support in Cassandra

### PR DESCRIPTION
**Pull request by mrbrowning**
This change is based off the SDK region awareness change which I made a small change to and am waiting for tests to pass again. Once that's merged, I'll rebase. The relevant changes here are only in the Cassandra framework.

### CODE REVIEW

File: frameworks/cassandra/universe/config.json [hunk 1]
```diff
@@ -83,6 +83,11 @@
           ],
           "default": "serial"
         },
+        "region": {
+          "description": "The name of the region that nodes in the Cassandra cluster should run in.",
```
Comment (gabrielhartmann) [inline frameworks/cassandra/universe/config.json]: Ending a sentence with a preposition

**Comment by gabrielhartmann** [comment]
Ideally we'll have some integration tests with this.  Still working on cluster generation with regions?

File: frameworks/cassandra/universe/config.json (hunk 1 above)
Comment (gabrielhartmann) [reply frameworks/cassandra/universe/config.json]:
```
The Cassandra cluster should run in this region.
```

File: frameworks/cassandra/universe/config.json [hunk 2]
```diff
@@ -83,6 +83,11 @@
           ],
           "default": "serial"
         },
+        "region": {
```
Comment (elezar) [inline frameworks/cassandra/universe/config.json]: From a customer UX perspective, would it make sense to have the "region" field and the "rack" and "data_center" fields adjacent to each other, as they both have to do with placement of the service. This is of course something that can be changed after the fact.

**Pull request merged**